tide_site.module

```diff
@@ -127,11 +127,11 @@ function tide_site_entity_presave(EntityInterface $entity) {
 }
 
 /**
- * Implements hook_field_widget_form_alter().
+ * Implements hook_field_widget_single_element_form_alter().
  *
  * @todo Extract into a service.
  */
-function tide_site_field_widget_form_alter(&$element, FormStateInterface $form_state, $context) {
+function tide_site_field_widget_single_element_form_alter(&$element, FormStateInterface $form_state, $context) {
```

Comment:
deprecated in 9.5 and removed in d10, so had to update this to work - https://www.drupal.org/node/3180429